hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/performance-considerations.adoc

```diff
@@ -66,7 +66,7 @@ emitting is rolling aggregation. Consider this example where we use
 include::{javasource}/Serialization.java[tag=modify-emitted]
 ----
 
-<1>  You keep emitting the same list, changing it in-place
+<1>  Here you both modify and emit the same list
```

Comment:
The previous version was clearer to me. What about:
> You should not continue to modify the item that you emit.

Reply:
This is just the first sentence of the full explanation that continues below.

Reply:
What about:
> Here you emit an object that's your mutable state.

In "Here you both modify and emit the same list" - we don't _modify_ it here.

Reply:
I updated the wording again, you can review.